src/parcels/_core/particlefile.py

```diff
@@ -309,7 +312,12 @@ def _convert_particle_data_time_to_float_seconds(particle_data, time_interval):
     #! Important that this is a shallow copy, so that updates to this propogate back to the original data
     particle_data = particle_data.copy()
 
-    particle_data["time"] = ((particle_data["time"] - time_interval.left) / np.timedelta64(1, "s")).astype(np.float64)
+    if time_interval is None:
+        particle_data["time"] = (particle_data["time"] / np.timedelta64(1, "s")).astype(np.float64)
```

Comment:
What is `particle_data["time"]` here? A timedelta object or a datetime object? (and if datetime, is that wrt. the beginning of the simuation?)